modules/localStorage.js

```diff
@@ -0,0 +1,35 @@
+import { addNewBook } from './addAndRemoveBook.js';
+
+export function loadFromLocalStorage() {
+  let Books;
+
+  if (localStorage.getItem('BooksInfo')) {
+    Books = JSON.parse(localStorage.getItem('BooksInfo'));
+  } else {
+    Books = [];
+  }
+  return Books;
+}
+
+export const displayBooksOnPage = () => {
+  const Books = loadFromLocalStorage();
+
+  Books.forEach((book) => {
+    addNewBook(book);
+  });
+};
+
+export function removeFromLocalStorage(element) {
+  let k = 0;
+  const Books = loadFromLocalStorage();
+  const idd = element.parentElement.id;
+  const newID = Number(idd);
+  for (let i = 0; i < Books.length; i += 1) {
+    if (Books[i].id === newID) {
+      k = i;
+      break;
+    }
+  }
+  Books.splice(k, 1);
+  localStorage.setItem('BooksInfo', JSON.stringify(Books));
+}
```

Comment:
- [ ] Kindly use arrow function syntax for all your functions as required. It's standard in ES6 to use arrow function syntax for your functions. Please modify your functions to arrow function syntax 👍🏽